.github/ISSUE_TEMPLATE/user-problem.md

```diff
@@ -18,7 +18,13 @@ Put the problem summary here.
 Diagnostic files
 ----------------
 
-If a crash (gpf, assert, abort, etc) or OutOfMemory condition, please provide the diagnostic files produced (javacore, Snap, jitdump, core). The smaller files can be attached to this Issue. The core should be compressed and made available via a file sharing service (Box, Google Drive, etc). If there are privacy concerns please direct email the files to an OpenJ9 committer.
+If a crash (gpf, assert, abort, etc) or OutOfMemory condition, please provide the diagnostic files produced (core, javacore, jitdump, Snap). The smaller files can be attached to this Issue.
+
+The system core, which is different from the javacore and a very important diagnostic, should be first processed with `jpackcore` (except on Windows), see https://eclipse.dev/openj9/docs/tool_jextract/#dump-extractor-jpackcore. The system core, which is likely quite large, should be compressed (`jpackcore` does this), and made available via a file sharing service (Box, Google Drive, OneDrive, etc). If there are privacy concerns please directly email or Slack the files or link/password to an OpenJ9 committer.
```

Comment:
"important diagnostic" -> "important source of diagnostic information"
"etc" -> "etc."